src/NAR/Constructs/SquareRootConstruct.java

```diff
@@ -85,16 +85,19 @@ public void execute() {
         return vals;
     }
 
-    public static void main(String args[]) {
-        String statement = "square root of 9";
-        Editor.run();
-        Parser.checkUserInput(statement);
-    }
+
 
 
     @Override
     public String getHelpInformation() {
-        return null;
+        return "Square Root:\n"
+                + "   Syntax: square root of <variable/number> "
+
+                + "\nPrints the square root of operand 1"
```

Comment:
"prints the square root of the provided operand" or replace operand by "variable/number"